bind/files/zone.jinja

```diff
@@ -3,6 +3,7 @@
 ; Modify the values passed to the bind pillar instead.
 ;
 {%- set zone_serial = soa['serial'] %}
+{%- set serial_auto = zone_serial == 'auto' %}
```

Comment:
This line is setting both `seria_auto` and `zone_serial` to *auto*, overwriting `zone_serial` (it wll always evaluate to 'auto').

Reply:
== is a comparison, not assigment, so this line does not change zone_serial

Reply:
you, right! I'm an idiot, I didn't realize. Please, forget about my comment.